src/admin/components/ResourceTabs/AuditLogTab/AuditLogTab.tsx

```diff
@@ -13,12 +13,15 @@ import SiteAuditLogEntityStatus from "./components/SiteAuditLogEntityStatus";
 import SiteAuditLogEntityStatusSide from "./components/SiteAuditLogEntityStatusSide";
 import SiteAuditLogProjectStatus from "./components/SiteAuditLogProjectStatus";
 import { AuditLogButtonStates } from "./constants/enum";
+import { AuditLogEntity } from "./constants/types";
 
 interface IProps extends Omit<TabProps, "label" | "children"> {
   label?: string;
+  isReport?: boolean;
+  reportEntityType?: AuditLogEntity;
 }
 
-const AuditLogTab: FC<IProps> = ({ label, ...rest }) => {
+const AuditLogTab: FC<IProps> = ({ label, isReport = false, reportEntityType, ...rest }) => {
   const { record, isLoading } = useShowContext();
   const [buttonToogle, setButtonToogle] = useState(() => {
     return record?.project ? AuditLogButtonStates.SITE : AuditLogButtonStates.PROJECT;
```

Comment:
You already have the context this component is rendered in with the `useShowContext` above. With that, you can determine if this is being shown on a nursery report page, site report, etc. Then this `buttonToggle` state should reflect that, and the `reportEntityType` addition to `useAuditLogActions` isn't needed either. Let me know if you'd like more direction than that.